apps/extension/src/state/wallets.ts

```diff
@@ -35,6 +38,26 @@ export const createWalletsSlice =
         await local.set('wallets', [newWallet.toJson(), ...wallets]);
         return newWallet;
       },
+      getSeedPhrase: async () => {
+        const passwordKey = get().password.key;
+        const key = await Key.fromJson(passwordKey!);
+
+        const encryptedSeedPhrase = get().wallets.all[0]?.custody.encryptedSeedPhrase;
```

Comment:
See bottom of this file. We should use `getActiveWallet()` instead of manually getting [0] wallet